### ISSUE TITLE
why TODO: loss is wrong , is this fix ?

### ISSUE THREAD

**Issue by LiaoZihZrong** [VaticanCameos99/knowledge-distillation-for-unet#2]
https://github.com/VaticanCameos99/knowledge-distillation-for-unet/blob/e64903ebcf89acee21fc3726948f01481dab580d/train.py#L55

Hello Coder:
i see the line 
`        #TODO: loss is wrong`
`        loss = loss_fn_kd(output, teacher_output, gt)`
**is this loss fix ?**

thanks all!
